girder/web_client/src/views/widgets/HierarchyWidget.js

```diff
@@ -294,6 +294,19 @@ var HierarchyWidget = View.extend({
             }
         }
 
+        if (this.parentModel.resourceName === 'collection' && this._showMetadata) {
+            if (!this.metadataWidget) {
```

Comment:
To be consistent with the folder initialization, should a function `_initCollectionViewSubwidgets` be added ?

Reply:
I actually don't like the logic in the block above this. It seems to condition display of the metadata widget on the value of `_showItems`, which seems like buggy behavior. Since it's just one widget in the collection case, I prefer to leave it as written.